src/mono/mono/utils/mono-time.c

```diff
@@ -307,22 +312,20 @@ mono_clock_get_time_ns (mono_clock_id_t clk_id)
 void
 mono_clock_init (mono_clock_id_t *clk_id)
 {
-	// TODO: need to implement this function for PC
-	g_assert_not_reached ();
+	// TODO: need to implement this function for Windows
```

Comment:
On the topic of todos, do we have an issue somewhere to clean up some ifdef usage? I definately spent a few minutes staring at this wondering "How do we have two functions with the same name/sig?" before I figured out the if defs.

Reply:
Our codebase is filled with them, mostly out of necessity. I'm not sure you can easily 'clean up' ifdef usage when you have so many build configurations and two entirely separate modes. If you have specific scenarios where you think it's valuable to switch from idefs to a more generalized solution I'm certainly open to discussion on it, and if you have specific instances with unnecessary duplication feel free to open an issue or PR.

Reply:
Yeah, I have some ideas, will discuss else where.